<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Traffic Generator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/img/TRAFFIC.ico</normaloff>:/img/TRAFFIC.ico</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>581</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/START.png</normaloff>:/img/START.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/STOP.png</normaloff>:/img/STOP.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="destinationDirLabel">
        <property name="text">
         <string>Папка назначения:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="destinationDirLE">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="destinationDirButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/FOLDER.png</normaloff>:/img/FOLDER.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="generationFileNbLabel">
        <property name="text">
         <string>Скорость генерации:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="generationFileNbSB">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="slashLabel">
        <property name="text">
         <string>файлов /</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="generationIntervalSB">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>3600</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="generationIntervalSecLabel">
        <property name="text">
         <string>сек.</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="infectedFileProbabilityLabel">
        <property name="text">
         <string>Вероятность генерации вируса:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="infectedFileProbabilityDSB">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="generationProgressLabel">
        <property name="text">
         <string>Сгенерировано:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="generationProgressInfoLabel">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="filesLabel">
        <property name="text">
         <string>файлов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string> объемом</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="totalVolumeInfoLabel">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="totalVolumeUnitCB">
        <item>
         <property name="text">
          <string>бит</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>байт</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Кбит</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>КБайт</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Мбит</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>МБайт</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Гбит</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ГБайт</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="virusNbLabel">
        <property name="text">
         <string>Вирусов:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="virusNbInfoLabel">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="currentSpeedLabel">
        <property name="text">
         <string>Скорость текущая:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="currentSpeedInfoLabel">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="currentSpeedUnitCB">
        <item>
         <property name="text">
          <string>бит/с</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>байт/с</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Кбит/с</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>КБайт/с</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Мбит/с</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>МБайт/с</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Гбит/с</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ГБайт/с</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="averageSpeedLabel">
        <property name="text">
         <string>, средняя:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="averageSpeedInfoLabel">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="averageSpeedUnitCB">
        <item>
         <property name="text">
          <string>бит/с</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>байт/с</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Кбит/с</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>КБайт/с</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Мбит/с</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>МБайт/с</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Гбит/с</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ГБайт/с</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QProgressBar" name="generationProgressBar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>7</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="sourceFilesLayout">
      <item>
       <layout class="QVBoxLayout" name="cleanFilesLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="cleanFilesListLabel">
            <property name="text">
             <string>Чистые файлы</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cleanFilesAddButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="img.qrc">
              <normaloff>:/img/ADD.jpg</normaloff>:/img/ADD.jpg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cleanFilesClearButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="img.qrc">
              <normaloff>:/img/CLEAR.png</normaloff>:/img/CLEAR.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="cleanFilesTableView"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="infectedFilesLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="infectedFilesListLabel">
            <property name="text">
             <string>Зараженные файлы</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="infectedFilesAddButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="img.qrc">
              <normaloff>:/img/ADD.jpg</normaloff>:/img/ADD.jpg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="infectedFilesClearButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="img.qrc">
              <normaloff>:/img/CLEAR.png</normaloff>:/img/CLEAR.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="infectedFilesTableView"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="workTimeLabel">
        <property name="text">
         <string>Время работы:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="workTimeInfoLabel">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
